tox.ini

```diff
@@ -72,7 +73,10 @@ passenv =
     {tests,functests}: PYTEST_ADDOPTS
     functests: BROKER_URL
 setenv =
+    PYTHONUNBUFFERED = 1
     dev: PYTHONPATH = .
+    dev: APP_URL = {env:APP_URL:http://localhost:5000}
+    dev: WEBSOCKET_URL = {env:WEBSOCKET_URL:ws://localhost:5001/ws}
```

Comment:
These two were previously set in `hypothesis devserver` and to different values for SSL and non-SSL. They're now set here with the non-SSL values as the defaults, but allowing them to be overridden by the external environment (and `make devssl` does so).